study-builder/studies/osteo/study.conf

```diff
@@ -1631,6 +1646,20 @@
       "dispatchToHousekeeping": false,
       "order": 3
     },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "FAMILY_HISTORY"
+      }
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 4
```

Comment:
Some triggers to create the family history parent activity. I don't know if this is where it belongs in the workflow at this point, but this gets family history created for now.

Reply:
Speaking of where family history belongs, have we considered how to "deliver" family history survey to existing participants? What I mean is, existing participants had already gone through the study workflow (e.g. completed consent, etc) so the events/triggers we add here won't apply to them. Are we going to manually create family history surveys for existing participants? Is this something study staff have considered yet?

Reply:
This is a very good question @yufengwng . I don't get sense that CMI folks have carefully considered where family history fits in the flow or what should be done about currently enrolled participants. I'll try to come up with some sharp/to the point questions for them regarding these issues.

Reply:
Inquired that this is a task that needs to be done. I have created Epic for this (anticipating there will be multiple tasks)
https://broadinstitute.atlassian.net/browse/DDP-5768